test-utils/generators/BasicGenerators.scala

```diff
@@ -213,13 +215,14 @@ trait BasicGenerators extends EitherValues {
     } yield Call(method, url)
   }
 
-  val paginationGen: Gen[Pagination] = {
+  val paginationGen: Gen[PaginatedViewModel] = {
     for {
+      label <- nonEmptyMessage
       currentPage <- intsBelowValue(5)
```

Comment:
intsBelowValue can reutrn a negative number, doesn't seem right for pagination

Reply:
I changed implementation of intsBelow to `Gen.choose(0, value)`